app/utilities/schema.py

```diff
@@ -177,7 +178,44 @@ def load_schema_from_url(schema_url, language_code):
 
     constructed_schema_url = f"{schema_url}?language={language_code}"
 
-    req = requests.get(constructed_schema_url)
+    session = requests.Session()
+
+    retries = Retry(
+        total=3,
+        allowed_methods=frozenset({"GET"}),
+        backoff_factor=0.1,
```

Comment:
The card specifies no back off. Since the timeout is 3 seconds, with a backoff of 0.1, what is the longer time before a user may see a response?.

Edit:
Looks like with 0.1 it would be `[0.0s, 0.2s, 0.4s, …]`. 

Since we may change total retries at any point, we should set `backoff_max` to 0.5 seconds.

Reply:
Overwriting default BACKOFF_MAX with value of 0.2, thanks!